pages/club.js

```diff
@@ -79,7 +70,9 @@ const Club = ({ query, userInfo, favorites, updateFavorites }) => {
 
 Club.getInitialProps = async props => {
   const { query } = props
-  return { query }
+  const resp = await doApiRequest(`/clubs/${query.club}/?format=json`)
```

Comment:
Not that we have it right now, but should error handling be happening here? what if we don't get a response / we get a 500?

Reply:
I think it'll throw an exception, which we won't catch and a 500 error (probably an ugly one) will be shown – I'll add a try/catch so that we show our error page.